cypress/locators/dropdown/index.js

```diff
@@ -0,0 +1,12 @@
+import { FORM } from '../locators';
+import {
+  DROPDOWN_INPUT, DROPDOWN_COMPONENT, DROPDOWN_LABEL,
+} from './locators';
+
+// knobs locators
+export const readOnlyCheckbox = () => cy.get(FORM).contains('readOnly').find('input');
```

Comment:
should we move readOnly to common staff ?

Reply:
readOnlyCheckbox is used only for Dropdown component, in my opinion current location is correct

Reply:
good point